spec/requests/unwithdraw_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+RSpec.describe "Unwithdraw" do
+  let(:edition) { create(:edition, :withdrawn, :political, :past_government) }
+
+  describe "POST /documents/:document/unwithdraw" do
+    context "when the edition is in history mode" do
+      context "and the user is a managing_editor" do
```

Comment:
Since we don't have a separate context can this be within the previous context?

Reply:
Context removed!